running locally/docker-prestissimo/docker-compose.yml

```diff
@@ -0,0 +1,119 @@
+version: '1'
```

Comment:
version too old

Reply:
Updated to 3.8 as you have in https://github.com/prestodb/prestorials/blob/main/docker-compose/docker-compose.yaml